FirebasePerformance/Tests/Unit/FPRAppActivityTrackerTest.m

```diff
@@ -193,4 +205,88 @@ - (void)testApplicationStateManagement {
   XCTAssertEqual(appTracker.applicationState, FPRApplicationStateBackground);
 }
 
+- (void)testIsApplicationPrewarmedReturnsYesBecauseOfDoubleDispatch {
+  id mockAppTracker = OCMPartialMock([FPRAppActivityTracker sharedInstance]);
+  OCMStub([mockAppTracker isPrewarmAvailable]).andReturn(YES);
+  OCMStub([mockAppTracker isDoubleDispatchEnabled]).andReturn(YES);
+  OCMStub([mockAppTracker isActivePrewarmEnabled]).andReturn(NO);
+
+  [FPRAppActivityTracker load];
```

Comment:
This should be called automatically. isn't it?

Reply:
this test actually fails if `+load()` isn't manually called. Somehow `appDidFinishLaunching` callback happens before `+load()` in the unit test.